### PR TITLE
Update migration guide for #1205

### PR DESCRIPTION
**Pull request by qinsoon**

### CODE REVIEW

**Comment by qinsoon** [comment]
binding-refs
JULIA_BINDING_REPO=qinsoon/mmtk-julia
JULIA_BINDING_REF=update-mmtk-core-1205

File: docs/userguide/src/migration/prefix.md [hunk 1]
```diff
@@ -32,6 +32,27 @@ Notes for the mmtk-core developers:
...
+API changes:
+
+-   module `memory_manager`
+    +   `handle_user_collection_request` now returns `bool` to indicate if a GC is triggered by the method.
+        Bindings may use the value, or simply ignore it.
+
+See also:
+
+-   PR: <https://github.com/mmtk/mmtk-core/issues/1205>
+-   Exampels:
```
Comment (wks) [inline docs/userguide/src/migration/prefix.md:52]:
```suggestion
-   Examples:
```

**Review by wks — CHANGES_REQUESTED** [review]
Just an obvious typo.  Good otherwise.

**Review by wks — APPROVED** [review]
LGTM

**Pull request merged**